.storybook/polyfills.js

```diff
@@ -2,18 +2,4 @@ import "core-js/es";
 
 import "zone.js/dist/zone";
 
-if (!Element.prototype.matches) {
-    Element.prototype.matches = Element.prototype.msMatchesSelector || Element.prototype.webkitMatchesSelector;
-}
-  
-  if (!Element.prototype.closest) {
-    Element.prototype.closest = function(s) {
-        var el = this;
-  
-        do {
-            if (el.matches(s)) return el;
-            el = el.parentElement || el.parentNode;
-        } while (el !== null && el.nodeType === 1);
-            return null;
-    };
-}
\ No newline at end of file
+import 'element-closest-polyfill';
```

Comment:
Works ... Be sure to respond to the issue directly letting them know they have to install a polyfill for `element.closest`